mnd/mnd/templates/rdrf_cdes/index.html

```diff
@@ -28,7 +28,7 @@
   </div>
   <div class="links d-none d-md-block">
     <div>
-      <a href="{% url 'two_factor:login' %}?next=/router/"
+      <a href="{% url 'two_factor:login' %}?next=/mnd/"
```

Comment:
Should be a reference to `{{registry.code}}` (which looks to be available in the template) instead of hardcoding `mnd`.